tailwind.config.js

```diff
@@ -57,6 +57,9 @@ module.exports = {
         "vh-1/3": "calc(100vh / 3)",
         "vh-1/4": "calc(100vh / 4)",
         "vh-1/5": "calc(100vh / 5)",
+      },
+      minWidth: {
+        "msg": "3.5rem"
```

Comment:
can you make it so that it is less specific for the message, so something like "3.5": "3.5rem" ?

Reply:
Wouldn't a "3.5" class not work because of the period? If not, then I suppose a "7/2" class would also work (even if it looks a little weird haha)